src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -1421,92 +1424,152 @@ bool OutputStateMachineEngine::s_ParseColorSpec(const std::wstring_view string,
                                                 DWORD& rgb) noexcept
 {
     bool foundRGB = false;
+    bool isSharpSignFormat = false;

```

Comment:
The name "sharp sign" is from XParseColor documentation. I didn't want to use `hashtag` because it's too... twittery.

Reply:
@skyline75489 [Octothorpe?😅](https://www.google.com/search?q=octothorpe)

Reply:
That’s a word I had to look it up in online dictionary. So thanks but no😝